src/celeritas/ext/detail/GeantPhysicsList.cc

```diff
@@ -286,6 +291,13 @@ void GeantPhysicsList::add_e_processes(G4ParticleDefinition* p)
 
         CELER_LOG(debug) << "Loaded Coulomb scattering with "
                             "G4eCoulombScatteringModel";
+        if (options_.msc == MscModelSelection::wentzel_vi
+            || options_.msc == MscModelSelection::urban_wentzel)
+        {
+            CELER_LOG(warning)
+                << "Coulomb scattering may be inconsistent with msc="
```

Comment:
I do not understand why the Coulomb scattering should match with msc as it is an independent discrete model.

Reply:
OK, I'd thought (see https://github.com/celeritas-project/celeritas/pull/705#discussion_r1152058662 ) that coulomb scattering covered the large-angle single Coulomb scattering, and WentzelVI took care of the rest, so they're only useful in tandem. From the physics manual:
> G4WentzelVIModel - is the default model for all changed particles [VNIU10][eal16][eal17] including electrons and positrons above 100 MeV, it is included in Physics List together with G4CoulombScattering process, which is responsible for large angle scattering;

Reply:
typo in the Geant4 manual: all `changed` particles --> all `charged` particles.  Anyway,  two models are orthogonal and not-complementary.  The Single Scattering is a discrete process/model based on the Coulomb differential cross section while the Urban/WenzelVI is an integral effect summed over the macroscopic length scale - so they are  not completing each other even though msc models will still apply their scattering for a given length by coulomb scattering if it is selected process up to the point (so, why there should be a minimal lambda for msc which technically avoid the redundant angular deviation).  Probably confusion comes from the accidental choice of the same low energy cut for WenzelVI and the Single Coulomb scattering (which is expensive at relatively low energy for most of materials).

Reply:
Got it. Thanks for the explanation!